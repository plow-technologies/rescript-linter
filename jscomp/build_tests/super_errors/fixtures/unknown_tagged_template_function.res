let res = tagg`| 5 Ã— 10 = ${5} |`