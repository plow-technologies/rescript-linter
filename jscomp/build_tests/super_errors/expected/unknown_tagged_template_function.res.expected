
  [1;31mWe've found a bug for you![0m
  [36m/.../fixtures/unknown_tagged_template_function.res[0m:[2m1:11-14[0m

  [1;31m1[0m [2mâ”‚[0m let res = [1;31mtagg[0m`| 5 Ã— 10 = ${5} |`

  The value tagg can't be found